docs/general/manage-tokens/faq.md

```diff
@@ -150,42 +161,126 @@ Here's a task for you:
+you can import the mnemonic directly into the Oasis Wallet.
+
+:::danger
+
+When you import your Ledger mnemonic to a software wallet, consider
+that mnemonic *potentially exposed/compromised*, i.e. not appropriate for a
+hardware wallet mnemonic anymore. If you use a new hardware wallet in the
+future, **never restore it from the mnemonic that was previously used by any
+software wallet!**
+
+:::
+
+If you used your Ledger with the Oasis Wallet Web, the Extension or the Oasis
+node, then you will need to use the [Oasis unmnemonic CLI tool] (64-bit binaries
+available for [Linux][unmnemonic-linux], [MacOS][unmnemonic-macos] and
+[Windows][unmnemonic-windows]).
+
+Open a terminal (on Windows run `cmd`), move to the corresponding folder and
+execute it. An interactive prompt will be shown to you:
+
+1. select the **Ledger** algorithm,
+2. enter the number of words in the mnemonic (typically **24**) and carefully
+   type them in one by line,
+3. enter the private key index (start with **0** and gradually increase it by 1,
+   if the resulting account does not contain any tokens),
+4. answer **Yes** for writing the keys to disk,
+5. enter the output directory where the file containing the Oasis private key
+   should be stored into (you can leave the default folder starting with
+   `wallet-export-`)
+
+For example:
+
+```
+  unmnemonic - Recover Oasis Network signing keys from mnemonics
+
+? Which algorithm does your wallet use Ledger
+ WARNING:
+
+  Entering your Ledger device mnemonic into any non-Leger device
+  can COMPROMISE THE SECURITY OF ALL ACCOUNTS TIED TO THE MNEMONIC.
+  Use of this tool is STRONGLY DISCOURAGED.
+
+? Have you read and understand the warning Yes
+? How many words is your mnemonic 24
+? Enter word 1 *****
+? Enter word 2 ****
+? Enter word 3 ****
+? Enter word 4 ******
+? Enter word 5 ****
+? Enter word 6 *****
+? Enter word 7 ****
+? Enter word 8 *******
+? Enter word 9 ******
+? Enter word 10 *****
+? Enter word 11 ***
+? Enter word 12 *****
+? Enter word 13 *****
+? Enter word 14 ****
+? Enter word 15 ****
+? Enter word 16 ******
+? Enter word 17 ****
+? Enter word 18 *****
+? Enter word 19 ****
+? Enter word 20 *******
+? Enter word 21 ******
+? Enter word 22 *****
+? Enter word 23 ***
+? Enter word 24 *****
+? Wallet index(es) (comma separated) 4
+ Index[4]: oasis1qqwkm23pyg638xvl2nu00frxhapusjjv8qhh3p77
+? Write the keys to disk Yes
+? Output directory /home/oa/tmp/wallet-export-2023-11-28
+ Index[4]: oasis1qqwkm23pyg638xvl2nu00frxhapusjjv8qhh3p77.private.pem - done
+Done writing wallet keys to disk, goodbye.
+```
+
+Move to the output directory. There you should see a `.private.pem` file
+containing the private key and named after the address it belongs to. Open it
+with a text editor and copy the Base64-encoded key between the
+`-----BEGIN ED25519 PRIVATE KEY-----` and `-----END ED25519 PRIVATE KEY-----`,
+and [paste it into the Oasis wallet][oasis-wallet-import-private-key].
```

Comment:
Couldn't one use the new `import-file` to do the import directly?

Reply:
Great idea! Added.